superset-frontend/src/dashboard/components/menu/HoverMenu.tsx

```diff
@@ -16,23 +16,22 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { RefObject } from 'react';
 import cx from 'classnames';
 
-const propTypes = {
-  position: PropTypes.oneOf(['left', 'top']),
-  innerRef: PropTypes.func,
-  children: PropTypes.node,
+interface HoverMenuProps {
+  position?: 'left' | 'top',
+  innerRef?: RefObject<any>,
```

Comment:
we're trying to avoid using `any`, maybe:
- See if you can find the type here
- See if `unknown` works

And apply this principle to all the other cases of `any` too, thanks!